Format.ts

```diff
@@ -0,0 +1,47 @@
+// ----- Types ----- //
+
+const enum Pillar {
+    News,
+    Opinion,
+    Sport,
+    Culture,
+    Lifestyle,
+}
+
+const enum Design {
+    Article,
+    Media,
+    Review,
+    Analysis,
+    Comment,
+    Feature,
+    Live,
+    Recipe,
+    MatchReport,
+    Interview,
+    GuardianView,
+    Quiz,
+    AdvertisementFeature,
+}
+
+const enum Display {
+    Standard,
+    Immersive,
+    Showcase,
+}
+
+interface Format {
+    pillar: Pillar;
+    design: Design;
+    display: Display;
+}
+
+
+// ----- Exports ----- //
+
+export {
+    Pillar,
+    Design,
+    Display,
+    Format,
+}
```

Comment:
This is a stylistic thing but I find exporting each item individually (in place) easier to reason about. I'm not obsessive about it though, what do you think?

Reply:
I find the opposite; I like `import`s at the top, `export`s at the bottom. I find it reduces the amount of syntax you have to parse in your head when reading the function, particularly now we have type annotations as well.

That said, I'm happy to settle on one or the other and stick to it.